### ISSUE TITLE
Incorrect definition for LRESULT

### ISSUE THREAD

**Issue by Plecra** [microsoft/windows-rs#974]
https://microsoft.github.io/windows-docs-rs/doc/bindings/Windows/Win32/Foundation/struct.LRESULT.html

> ```rust
> #[repr(transparent)]
> pub struct LRESULT(pub i32);
> ```

https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#LRESULT

> This type is declared in WinDef.h as follows:
> `typedef LONG_PTR LRESULT;`

The definition generated by windows-rs should be `struct LRESULT(pub isize);`

I encountered this issue when trying to [return an error from `WndProc`](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-create#return-value) - `-1` winds up looking like `-4294967296` (`(-1i32 as i64) << 32`) to the system

**Comment by kennykerr** [comment]
Agreed, that looks wrong. Transferring to the win32metadata repo, where this definition comes from.
